packages/flutter/lib/src/material/bottom_app_bar.dart

```diff
@@ -131,16 +130,18 @@ class _BottomAppBarState extends State<BottomAppBar> {
     final bool hasFab = Scaffold.of(context).hasFloatingActionButton;
     final NotchedShape? notchedShape = widget.shape ?? babTheme.shape;
     final CustomClipper<Path> clipper = notchedShape != null && hasFab
-      ? _BottomAppBarClipper(
-          geometry: geometryListenable,
-          shape: notchedShape,
-          materialKey: materialKey,
-          notchMargin: widget.notchMargin,
-        )
-      : const ShapeBorderClipper(shape: RoundedRectangleBorder());
-    final double elevation = widget.elevation ?? babTheme.elevation ?? _defaultElevation;
-    final Color color = widget.color ?? babTheme.color ?? Theme.of(context).bottomAppBarColor;
-    final Color effectiveColor = ElevationOverlay.applyOverlay(context, color, elevation);
+        ? _BottomAppBarClipper(
```

Comment:
The framework doesn't use `dart format`, so you'll have to turn it off for the workspace if you're using vs code..sorry

```
{
    "editor.formatOnSave": false,
    "editor.formatOnPaste": false
}
```